ARImageTrackingExamples/ARImageTrackingExamples/Helpers/UIViewController+helpers.swift

```diff
@@ -9,8 +9,15 @@ extension UIViewController {
 
 extension Float {
     static let smallOffset: Float = 0.01
+    static let rightAngle: Float = .pi / 2
 }
 
 extension CGFloat {
     static let slightyTransparent: CGFloat = 0.95
+    static let half: CGFloat = 0.5
+    static let third: CGFloat = 0.75
```

Comment:
that's not a third 😛

Reply:
@amlcurran You're right its not a .`third` I also noticed we have other consts like `static let slightyTransparent: CGFloat = 0.95` and `static let smallOffset: Float = 0.01`. I was understand it is to get rid of magical number, but Im not sure if in that case it makes the code more readable and just wanted to know your opinion.

Reply:
If you name your constant correctly it's like a good comment, with intent included.

Reply:
@malaba I agree. However I think in our situation when you're setting transparency in code ` planeNode.opacity = 0.95` seems to be readable enough especially if this value is either given by designers or chosen by you because thats how you felt. In my opinion adding even the best descriptive constant is redundant. Of course the situation would be different if that would be a part of colour theme or something, but thats a different discussion.

Reply:
@amlcurran in that case shouldn't we move `.slightyTransparent` to a function that it is used in? Other wise it seems like it may be used in several places in which `. slightyTransparent` means something different. If we do that should we also create additional consts for zeros in code: 
` let webView = UIWebView(frame: CGRect(x: 0,
                                                  y: 0,
                                                  width: cardSize.width,
                                                  height: cardSize.height))`
Don't we risk to end up in a situation where our function have more code connected to declaring consts then the actual functionality? 
Where would you say is a line in a situation like that ?

Reply:
@amlcurran 

> well yeah if you're going to go as extreme as the example you suggested, but would you actually do that or are you just trying to make a point?

Not to make a point, in my head those were equally extreme situations, but now, I think I can see how you look at it. Thanks I will definitely look at your H&T.

Reply:
awesome :)